update keys

JohnnyJumper opened the pull request:
`3c25f57353234b1b853e9861050f4817` -> this key is secured and setup to whitelist all UD contracts only
`e05c36b6b2134ccc9f2594ddff94c136` -> this key is not secured and just a placeholder until we find a solution for ens. We can't whitelist all resolvers since there are a lot of them.

DeRain commented on Sources/UnstoppableDomainsResolution/Resolution.swift:
```diff
@@ -58,7 +58,11 @@ public class Resolution {
         do {
-            networkServices.append(try ENS(network: network, providerUrl: providerUrl, networking: networking))
+            var ensProviderUrl = providerUrl
+            if providerUrl == "https://mainnet.infura.io/v3/3c25f57353234b1b853e9861050f4817" {
```
@JohnnyJumper It's not a good idea to have condition like this. We should be able to set service provider explicitly: a different endpoints for CNS, ENS, and ZNS

DeRain reviewed (changes requested):
@JohnnyJumper Make sure CI is running and README updated accordingly. We want to say explicitly, that we're using Infura without rate limits restrictions for CNS.

For ENS resolution it's recommended to setup own provider since we don't guarantee ENS Infura key availability.

JohnnyJumper commented:
Added an ability to configure each naming service separately by providing one struct `Configurations` with fields:
```swift
public struct NamingServiceConfig {
    let network: String
    let providerUrl: String
    let networking: NetworkingLayer
}

public struct Configurations {
    let cns: NamingServiceConfig
    let zns: NamingServiceConfig
    let ens: NamingServiceConfig
    }
```

It has been configured by default to use our infura keys for CNS and ENS, as well as to use Zilliqa API `https://api.zilliqa.com/`

I have preserved the old constructor as this would be a breaking change for users. We should not forget to remove it when we release a new major version. Also added a test that makes sure nothing breaks with old usage.

I couldn't found an appropriate way to declare a constructor as deprecated, @merenkoff does swift has something useful for this case?

Another note: Resolution class had a private property `providerUrl` which hasn't been used anywhere besides the initial configurations. I have removed it from the class fields, but left it in the old constructor as it was one of the params;

Updated Readme and comments accordingly
